Mathlib/Geometry/Manifold/Notation.lean

```diff
@@ -329,6 +333,47 @@ where
         | _ => return none
       | throwError "Couldn't find a `ModelWithCorners` with model space `{H}` in the local context."
     return m
+  /-- Attempt to find a model with corners on a space of continuous linear maps -/
+  -- TODO: should this also support continuous linear equivalences?
+  fromCLM : TermElabM Expr := do
+    match_expr e with
+    | ContinuousLinearMap k S _ _ _σ _E _ _ _F _ _  _ _ =>
+      if ← isDefEq k S then
```

Comment:
Hmm, I want to be sure that `isDefEq` at this transparency doesn't make instance synthesis break. Can you write a test for this and for the one in `fromRealInterval` where `isDefEq` nontrivially succeeds (but `withReducible <| isDefEq` wouldn't)? E.g., wrapping in a `def`, such as using `def RealCopy := Real` for the `fromRealInterval` test.

Reply:
Great! Also, if `isDefEq` turns out to be fine, we should document that we know it's fine near its invocation in a short comment. :)

Reply:
I also pushed a test for the continuousLinearMap case; this already breaks earlier (at the synthesis of a NormedSpace instance for `E →L[𝕜] E''` :thinking: